tests/base.py

```diff
@@ -49,10 +49,8 @@ class BrowseClass(BrowseView):
             columns = [
                 ("Name", "name"),
                 ("Text", "other__text"),
-                (
-                    "Model1",
-                    "model1",
-                ),
+                ("Model1", "model1"),
+                ("Roundabout Name", "get_name"),
```

Comment:
Including a non-field attr like `get_name` triggers a bug arising from changing behavior in Django 3.1.13